va/va_test.go

```diff
@@ -340,9 +340,10 @@ func TestHTTPTimeout(t *testing.T) {
 	started := time.Now()
 	_, prob := va.validateHTTP01(ctx, dnsi("localhost"), chall)
 	took := time.Since(started)
-	// Check that the HTTP connection times out after 5 seconds and doesn't block for 10 seconds
-	test.Assert(t, (took > (time.Second * 5)), "HTTP timed out before 5 seconds")
-	test.Assert(t, (took < (time.Second * 10)), "HTTP connection didn't timeout after 5 seconds")
+	// Check that the HTTP connection does't return before a timeout, and times
+	// out after 10 seconds
+	test.Assert(t, (took > (time.Second * 10)), "HTTP timed out before 10 seconds")
```

Comment:
Could you make these all use `singleDialTimeout` so if we change it again we don't have to update the tests.

Reply:
Good call :+1: - Fixed in 3ad8494